core/embed/extmod/modtrezorui/mipi_lcd.c

```
@@ -2,727 +2,809 @@
 
 #include "mipi_lcd.h"
 
+static int dummy_dbg_printf(const char* fmt, ...)
+{
+    return 0;
+}
+
+static DbgPrintf_t dbg_printf = dummy_dbg_printf;
+
+inline void disp_set_dbg_printf(DbgPrintf_t func)
```

cppcheck (2.10-2)

[style] The function 'disp_set_dbg_printf' is never used.

(unusedFunction)